packages/lib/BaseModel.ts

```diff
@@ -53,6 +53,8 @@ export interface DeleteOptions {
 	// parent. By default the item will be moved at the root of the trash. Note
 	// that caller must ensure that this parent ID is a deleted folder.
 	toTrashParentId?: string;
+
+	shouldDeleteRevisions?: boolean;
```

Comment:
Please remove the option, that's too much custom code just to support one line in a test. Please put back the SQL then